kubernetes-client/src/test/java/io/fabric8/kubernetes/TestUtils.java

```diff
@@ -0,0 +1,20 @@
+package io.fabric8.kubernetes;
```

Comment:
license header missing here

Reply:
After looking at this patch closely, I think it would be better to move this method to `Utils` instead of creating a separate `TestUtils`